src/reynir_correct/errfinder.py

```diff
@@ -207,6 +208,12 @@ def verb_subject_matches(cls, verb: str, subj: str) -> bool:
             verb, set()
         ) or subj in cls._VERB_ERROR_SUBJECTS.get(verb, dict())
 
+    @classmethod
+    def verb_matches_arguments(cls, key: str) -> bool:
+        """ Return True if the given arguments are allowed for this verb 
+        or if these are erroneous arguments which we can flag """
+        # This is overridden in reynir_correct.checker
```

Comment:
Is this comment correct (also in GreynirPackage), since this is actually errfinder.py?

Reply:
...and how is an erroneous verb frame then detected and reported to the user as an annotation?

Reply:
Copied from similar files in GreynirPackage, fixed.
This ensures erroneous verb frames are allowed in the parsing process. Without this, sentences such as 'Ég rústa bílnum' are not parsable. 
The error is then detected in check_obj() under _annotate_verb() in errfinder.py, and then annotated with a call to annotate_wrong_obj_case().